tests/cunit/CMakeLists.txt

```diff
@@ -74,13 +74,17 @@ if (NOT PIO_USE_MPISERIAL)
   add_executable (test_darray_1d EXCLUDE_FROM_ALL test_darray_1d.c test_common.c)
   target_link_libraries (test_darray_1d pioc)  
   add_executable (test_darray_3d EXCLUDE_FROM_ALL test_darray_3d.c test_common.c)
-  target_link_libraries (test_darray_3d pioc)  
+  target_link_libraries (test_darray_3d pioc)
   add_executable (test_decomp_uneven EXCLUDE_FROM_ALL test_decomp_uneven.c test_common.c)
   target_link_libraries (test_decomp_uneven pioc)  
   add_executable (test_decomps EXCLUDE_FROM_ALL test_decomps.c test_common.c)
   target_link_libraries (test_decomps pioc)
   add_executable (test_rearr EXCLUDE_FROM_ALL test_rearr.c test_common.c)
   target_link_libraries (test_rearr pioc)
+  if (PIO_USE_MALLOC)
```

Comment:
Yes, something about bget breaks for async. Probably it is not initializing buffers correctly.